packages/snaps-utils/src/types.ts

```diff
@@ -119,7 +117,7 @@ type UriOptions<Type extends string> = {
 };
 
 export const uri = (opts: UriOptions<any> = {}) =>
-  refine(union([string(), instance(URL)]), 'uri', (value) => {
```

Comment:
What's the reason for changing this? Technically this is breaking 🤔

Reply:
Because it doesn't make any sense to validate JSON-RPC parameters against instances, we don't use it for this and it only made the types incompatible with JSON-RPC 😓